medleydb/annotate/generate_activations_annotations.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import division
+import scipy.signal
+import numpy as np
+import librosa
+import medleydb
+import os
+import argparse
+
+
+def create_activation_annotation(
+    mtrack,
+    win_len=2048,
+    lpf_cutoff=0.075,
+    theta=0.15,
+    binarize=False
+):
+
+    H = []
+
+    for track_id, track in mtrack.stems.items():
+        audio, rate = librosa.load(track.file_path, mono=False)
```

Comment:
the time vectors were indicating otherwise, but I now found that it's actually the window length parameter that need to be changed. So `sr=441000` and `win_length=4096` seems to match.

Results are still a bit off. I am working on it...

```
time,S01,S02,S03,S04,S05
0.0000,0.9927,0.6062,0.1231,0.5787,0.8933
0.0464,0.9966,0.9341,0.3208,0.8458,0.9880
0.0929,0.9983,0.9919,0.6236,0.9549,0.9987
0.1393,0.9991,0.9989,0.8572,0.9870,0.9998
0.1858,0.9994,0.9998,0.9567,0.9959,1.0000
0.2322,0.9996,1.0000,0.9877,0.9985,1.0000
0.2786,0.9997,1.0000,0.9964,0.9994,1.0000
0.3251,0.9997,1.0000,0.9988,0.9997,1.0000
0.3715,0.9998,1.0000,0.9996,0.9998,1.0000
``` 

vs

```
time,S01,S02,S03,S04,S05
0.0000,0.9932,0.5501,0.0728,0.5285,0.8885
0.0464,0.9967,0.9246,0.2522,0.8291,0.9870
0.0929,0.9983,0.9913,0.5889,0.9523,0.9985
0.1393,0.9990,0.9989,0.8569,0.9868,0.9998
0.1858,0.9993,0.9998,0.9607,0.9960,1.0000
0.2322,0.9995,1.0000,0.9896,0.9986,1.0000
0.2786,0.9996,1.0000,0.9971,0.9994,1.0000
0.3251,0.9997,1.0000,0.9991,0.9997,1.0000
0.3715,0.9997,1.0000,0.9997,0.9999,1.0000
```

seems that the framing might still be an issue...